src/result_readers.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import re
+from itertools import groupby
+from rdflib import Graph, URIRef, SKOS
+
+
+class RegistryUrls:
+    NATIONAL = "data.brreg.no"
+    GEO_NORGE = "register.geonorge.no"
+
+
+class ParsedContent:
+    def __init__(self, count: int, org_id: str, name: str):
+        self.count = count
+        self.name = name
+        if not org_id.startswith("http"):
+            self.org_id = org_id
+
+        elif re.findall(RegistryUrls.NATIONAL, org_id):
+            self.norwegianRegistry_iri = org_id
+            self.org_id = self.get_norwegian_registry_id()
+        else:
+            if re.findall(RegistryUrls.GEO_NORGE, org_id).__len__() < 1:
+                logging.error(f"organization IRI {org_id} does not belong to any known registry")
+            self.alternativeRegistry_iri = org_id
+
+    def get_norwegian_registry_id(self):
+        if hasattr(self, 'norwegianRegistry_iri'):
```

Comment:
Instead of `hasattr` which clutters up the code since you have to repeatedly use it on every occasion, wrap the required method part try with except on `AtributeError`. Why? Because this way you will have control over errors and be able to log them accordingly without having to write hasattr every time and indent for an else/elif case.